src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carCostPerDay = 40;
+  const mediumRentTermDiscount = 20;
+  const longRentTermDiscount = 50;
+  const price = carCostPerDay * days;
+
+  if (days < 3) {
+    return price;
+  }
+
+  if (days < 7) {
```

Comment:
Let's add 3 and 7 to constants and  just a heads-up that it's a good practice in JavaScript to use SNAKE_CASE (all uppercase) for constant variables.
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/46566190/17912edd-4fa4-41ed-858e-8ed98ec12c10)